tests/ZendTest/Db/Adapter/Driver/Pgsql/ConnectionTest.php

```diff
@@ -86,9 +86,14 @@ public function testGetConnectionStringEncodeSpecialSymbol()
      */
     public function testSetConnectionTypeException()
     {
+        if (version_compare(PHP_VERSION, '5.6', 'lt')) {
```

Comment:
there are <code>if (PHP_VERSION_ID) </code> check used instead of <code>version_compare</code> in other source, for example : https://github.com/zendframework/zf2/blob/master/library/Zend/Stdlib/JsonSerializable.php#L12. 

what we will be using for consistencies ?